packages/core/src/xr/feature/XRPlatformFeature.ts

```diff
@@ -0,0 +1,46 @@
+import { IXRPlatformFeature, IXRFeatureDescriptor } from "@galacean/engine-design";
+import { Logger } from "../../base";
+import { Engine } from "../../Engine";
+
+/**
+ * The base class of XR platform feature.
+ */
+export abstract class XRPlatformFeature implements IXRPlatformFeature {
+  protected _engine: Engine;
+
+  constructor(engine: Engine) {
+    this._engine = engine;
+  }
+
+  _isSupported(descriptor: IXRFeatureDescriptor): Promise<void> {
+    return Promise.resolve();
+  }
+
+  _initialize(descriptor?: IXRFeatureDescriptor): Promise<void> {
+    return Promise.resolve();
+  }
+
+  _onUpdate(): void {
```

Comment:
abstract class should not implement function

Reply:
Done.